hack/bin/BUILD.bazel

```diff
@@ -58,6 +58,18 @@ genrule(
     visibility = ["//visibility:public"],
 )
 
+genrule(
+    name = "fetch_oc3",
+    srcs = select({
+        ":darwin": ["@oc_3_11_darwin//:file"],
```

Comment:
I think we worked out that it does not work on darwin? Probably best to remove all traces of support so as to not mislead people. We can add it back if we can get it working :)